apexsteps.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Don't think your git ignore actually ingored this, probably just need to add some wildcards.